release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,55 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+- Due to Azure limitations, upgraded clusters do not support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
+
+---
+
+### azure-operator v2.9.0
+- **TODO**
+
+### Kubernetes v1.16.3
+- No change from the previous version
+
+### Calico v3.10.1
+- No change from the previous version
+
+### cert-exporter [v1.2.1](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#121-2019-12-24)
+- No change from the previous version
+
+### chart-operator [v0.11.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.11.4)
+- Now handling timeouts by pulling chart tarballs.
+
+### cluster-operator [v0.21.4](https://github.com/giantswarm/cluster-operator/releases/tag/v0.21.4)
```

Comment:
@axbarsan [azure 11.1.0 release](https://github.com/giantswarm/releases/blob/master/azure.yaml#L1-L15) currently references cluster-operator 0.22.1-dev.
I'd like to release that dev/wip cluster-operator legacy, 0.22.1, update azure 11.1.0 release to reference cluster-operator 0.22.1 instead of 0.22.1-dev

Then these release notes have to be updated, cluster-operator version from 0.21.4 to 0.22.1, and nginx-ingress-controller, just link text from Giant Swarm app v1.1.1 to Giant Swarm app v1.3.0

Reply:
Thank you, @sslavic 🚀